src/main/java/com/mindera/school/spaceshiprent/dto/spaceship/CreateOrUpdateSpaceshipDto.java

```diff
@@ -1,12 +1,14 @@
 package com.mindera.school.spaceshiprent.dto.spaceship;
 
+import lombok.Builder;
 import lombok.Data;
 
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Positive;
 
 @Data
+@Builder
```

Comment:
You can do this, cause it's not that much implementation, but we usually don't change the actual code to fulfill tests' needs. Not an issue, but keep in mind.